js/apps/system/_admin/aardvark/APP/frontend/js/views/queryView.js

```diff
@@ -2054,7 +2054,7 @@
             // fallback
             if (result.fallback && (result.fallback === 'geo' || result.fallback === 'geotable')) {
               $('#geo-switch').addClass('disabled').css('display', 'inline').css('opacity', '0.5');
-              $('#geo-switch').addClass('tippy').attr('title', 'No internet connection. Map is not available.');
+              $('#geo-switch').addClass('tippy').attr('title', 'No internet collection. Map is not available.');
```

Comment:
This seems to be a bad copy/paste error.

Reply:
wt...?  how did this change?